languages/python/oso/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
For some reason this is needed for pip versions >= 21.

It's something to do with them changing builds to happen in the same folder instead of a temporary dir. I don't know why this works, it just did.